scripts/artifacts-building/containers/build-process.sh

```diff
@@ -140,10 +140,10 @@ openstack-ansible containers/artifact-build-chroot.yml \
 
 # Build the list of roles to build containers for
 role_list=""
-role_list="${role_list} elasticsearch kibana rpc-role-logstash memcached_server"
-role_list="${role_list} os_cinder os_glance os_heat os_horizon os_ironic"
-role_list="${role_list} os_keystone os_neutron os_nova os_swift os_tempest"
-role_list="${role_list} rabbitmq_server repo_server rsyslog_server"
+role_list="${role_list} ansible-elasticsearch memcached_server os_cinder os_glance"
```

Comment:
The ansible-elasticsearch role does not use the same tagging conventions that we use, so artifacting the container in the same way that we do our other roles simply won't work correctly. To implementing an artifacted container for the role it will likely require a more specialised process - perhaps a custom playbook, or at the very least its own entry in the bash script to skip/include tags appropriately. Another option is to skip the container artifacting for elasticsearch so that it falls back to using the default container artifact and the apt artifacts at deploy time.